pkg/lvm/lvm2app_test.go

```diff
@@ -166,44 +109,20 @@ func TestLookupPhysicalVolumeNonExistent(t *testing.T) {
 }
 
 func TestValidateTag(t *testing.T) {
-	handle, err := NewLibHandle()
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer handle.Close()
-	if err := handle.ValidateTag(""); err != ErrTagInvalidLength {
-		t.Fatalf("Expected tag to fail validation")
-	}
-	if err := handle.ValidateTag(strings.Repeat("a", 1025)); err != ErrTagInvalidLength {
+	if err := ValidateTag(strings.Repeat("a", 1025)); err != ErrTagInvalidLength {
 		t.Fatalf("Expected tag to fail validation")
 	}
-	if err := handle.ValidateTag(strings.Repeat("a", 1)); err != nil {
+	if err := ValidateTag(strings.Repeat("a", 1)); err != nil {
 		t.Fatalf("Expected tag to pass validation")
 	}
-	if err := handle.ValidateTag(strings.Repeat("a", 1024)); err != nil {
+	if err := ValidateTag(strings.Repeat("a", 1024)); err != nil {
 		t.Fatalf("Expected tag to pass validation")
 	}
-	if err := handle.ValidateTag("some:tag"); err != ErrTagHasInvalidChars {
+	if err := ValidateTag("some:tag"); err != ErrTagHasInvalidChars {
 		t.Fatalf("Expected tag to fail validation")
 	}
 }
 
-func TestValidateVolumeGroupName(t *testing.T) {
```

Comment:
why did we delete the test for vg name validation?

Reply:
That function tested an internal function that was removed. There is a separate test that checks the 'external' API for when a volume group name is bad called `TestCreateVolumeGroupInvalidName`.